std-bits/table/src/main/java/org/enso/table/formatting/DateTimeFormatter.java

```diff
@@ -10,7 +11,9 @@ public class DateTimeFormatter implements DataFormatter {
   private final java.time.format.DateTimeFormatter formatter;
 
   public DateTimeFormatter(String formatString, Locale locale) {
-    formatter = java.time.format.DateTimeFormatter.ofPattern(formatString, locale);
+    formatter = formatString.equals("ENSO_ZONED_DATE_TIME")
+            ? Time_Utils.default_output_date_time_formatter()
+            : Time_Utils.make_formatter(formatString, locale);
```

Comment:
minor nitpick: I don't like that this constant `"ENSO_ZONED_DATE_TIME"` is placed here.

Maybe we can move this whole logic to `Time_Utils` instead?

Maybe we could make a:

```
Time_Utils {
	DateFormatter make_output_formatter(String formatString, Locale locale) {
		return formatString.equals("ENSO_ZONED_DATE_TIME")
	            ? default_output_date_time_formatter()
	            : make_formatter(formatString, locale);
	}
}
```